ReviewsReceiver/src/controllers/reviewsController.ts

```diff
@@ -0,0 +1,23 @@
+import reviewsModel, { IReview } from "../models/reviewsModel";
+
+class ReviewsController {
+  async post(req, res) {
+    let { businessId } = req.body;
+    let { reviews } = req.body;
+    reviews = reviews.map((review) => {
+      return {
+        ...review,
+        businessId,
+      };
+    });
```

Comment:
can immediately return without `{}` function
```suggestion
    reviews = reviews.map((review) => ({
        ...review,
        businessId,
    }));
```